sfs/mono/drivingfunction.py

```diff
@@ -419,61 +765,236 @@ def nfchoa_25d_plane(omega, x0, r0, n=[0, 1, 0], max_order=None, c=None):
     return 2*1j / r0 * d, selection, secondary_source_point(omega, c)
 
 
+def _max_order_circular_harmonics(N, max_order):
```

Comment:
This function is dead code and should be removed.

The function now lives here:

https://github.com/sfstoolbox/sfs-python/blob/8bb6387a7078c3ce654cd3790fe41b808230098f/sfs/util.py#L621-L645

Reply:
removed now, was not 100% sure about it.

Reply:
The tests pass, so it should be fine!